core/oiostr.h

```diff
@@ -87,24 +87,21 @@ void oio_str_randomize(gchar *d, const gsize dlen, const char *set);
 /** Fills 'b' with 'blen' random bytes */
 void oio_buf_randomize(guint8 *b, gsize blen);
 
-struct oio_str_autocontainer_config_s {
-	gsize src_offset;
-	gsize src_size;
-	gsize dst_bits;
-};
-
 /** Fills 'dst' with the name of the container deduced from the given 'path'.
  * 'dst' must be at least 65 characters long. */
```

Comment:
It's not 65 anymore. It's `(bits - 1) / 4 + 2` (4 bits by hex digit, one extra byte for the null character).